tools/data-handler/src/interfaces/project-interfaces.ts

```diff
@@ -205,8 +202,8 @@ export interface attachmentDetails {
   mimeType: string | null;
 }
 
-// Name for a card (consists of prefix and running number; e.g. 'test_1')
-export const cardNameRegEx = new RegExp(/^[a-z]+_[0-9]+$/);
+// Name for a card (consists of prefix and a random 8-character base36 string; e.g. 'test_abcd1234')
```

Comment:
Should you remove the `rollback()` function as well? It is probably no longer needed, as there is nothing to rollback for.

Reply:
Also `commit()` could be removed, and whenever the settings need to be saved, just call `persistConfiguration()` (rename to `save()`).